common/models/role.js

```diff
@@ -97,7 +96,7 @@ module.exports = function(Role) {
       }
 
       roleModel.roleMappingModel.find({
-        where: { roleId: this.id, principalType: principalType },
+        where: {roleId: roleModel.roleMappingModel.id, principalType: principalType},
```

Comment:
Thanks for landing this .. I will rebase and hopefully it will fix it in the next commit